overlay/lower/root/.ssh/id_dropbear.pub

```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAd7d9G2bYhoghDRxM+4FCWcc+sZg7G4r4LfwkbA1cda root@thingino-t31x
```

Comment:
It is better to create ssh key at the first boot instead of adding fixed ssh key.

Reply:
Fix this by the next force push